src/main/java/org/spdx/sbom/gradle/utils/SpdxDocumentBuilder.java

```diff
@@ -318,6 +318,10 @@ private Optional<SpdxPackage> createMavenModulePackage(
     if (dependencyFile != null) {
       ModuleVersionIdentifier moduleId = resolvedComponentResult.getModuleVersion();
       PomInfo pomInfo = poms.get(resolvedComponentResult.getId().getDisplayName());
+      if (pomInfo == null) {
```

Comment:
how are ivy dependency trees built? Are they explicitly defined by the consumer?

Reply:
Totally makes sense.
Yes, Ivy modules have a metadata file that can be inspected (similar to pom.xml), here is an example: https://repository.microej.com/modules/ej/library/ui/widget/5.2.1/ivy-5.2.1.xml So it is feasible to create such a module.

This issue is not really blocker (for the moment) so I can try to have a look to support Ivy module instead of ignoring them. I let you know.

Reply:
I think we can submit a version of this with the flag in SpdxSbomExtension to unblock builds.

Reply:
I pushed an update to add the property `ignoreNonMavenDependencies`. Let me know if it is ok for you.